migrations/Version202203021737164734_taoBackOffice.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoBackOffice\migrations;
+
+use common_report_Report as Report;
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\model\user\TaoRoles;
+use oat\tao\model\accessControl\func\AclProxy;
+use oat\tao\model\accessControl\func\AccessRule;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoBackOffice\controller\Lists;
+use oat\taoItems\model\user\TaoItemsRoles;
+
+final class Version202203021737164743_taoBackOffice extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Update global UI config';
+    }
+
+    const SHOW_OPTION = 'show';
+    const HIDE_OPTION = 'hide';
+    const GLOBAL_UI_CONFIG_NAME = 'globalUIConfig';
+
+    const DEFAULT_CONFIG = [
```

Comment:
Do you plan to replace this constant with the real config when TR-1978 is done? If not, here doesn't seem a good place to keep it. We should have it as a file under `config/default/`.

Reply:
moved (but I'm not sure that I'm reading from the default config in a right way now)